project_one.html

```diff
@@ -15,10 +15,84 @@
   </head>
   <body>
     <section class="container">
+      <div class="pageone">
+      <header class="header">
+        <nav class="logo">
+          <a href="#" class="logo1"> My Logo </a>
+          <ul>
+            <li> <a href="#">Portfolio</a> </li>
+            <li> <a href="#">About</a> </li>
+            <li> <a href="#">Contact</a> </li>
+          </ul>
+          
+          <div class="menu"><img src="./icons/Menu.png" alt="" /></div>
+        </nav>
+      </header>
+
+      <section class="content">
+        <div class="content_set" >
+        <div>
+          <h1 class="heading">I'm Saif <br> Glad to see you!</h1>
+        </div>
+        <div>
+ 
+            <p class="para_text">
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hestiate to contact me.
+            </p>
+       
+
+          <span class="extra">
+            <a href="#" class="connect"> LET'S CONNECT </a></span
+          >
+          <ul >
+            <li class="icons">
+              <a href="#" class="social">
+                <img
+                  src="icons/twitter.png"
+                  alt="Twitter"
+                  class="twitter"
+                />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img
+                  src="icons/Linkedin.png"
+                  alt="Linkedin"
+                  class="linkdein"
+                />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img src="icons/m.png" alt="M" class="m" />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img src="icons/github.png" alt="Github" class="git" />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img src="icons/peace.png" alt="peace" class="peace" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      </section>
+      </div>
       <div class="card">
         <div class="snap">
           <img src="./images/Snap.png" alt="img" />
         </div>
+        <div class="websnap">
+          <img src="./Portoflio Card/Snapshoot1.png" alt="">
+        </div>
         <section class="main">
           <h2>Tonic</h2>
```

Comment:
- [ ] Below image shows what your page looks like when viewed in a medium screen. Kindly ensure that the page is fully responsive across multiple screens and kindly remove that empty space to the right of the page

![image](https://user-images.githubusercontent.com/43995830/160108396-b3b98503-9fe8-4d79-98e0-42f5aad8cde1.png)